aip/general/0009.md

```diff
@@ -120,14 +120,18 @@ A Network API exposed by a Google service. Most of these are hosted on the
 `googleapis.com` domain. It does not include other types of APIs, such as client
 libraries and SDKs.
 
-### IaC
+### Declarative Clients
 
-Short for Infrastructure-as-code, IaC describes a category of clients that
+Declarative Clients describes a category of clients that
 consumes a markup language or code that represents resources exposed by an API,
 and executes the appropriate imperative actions to drive the resource to that
-desired state.
+desired state. To determine if a set of updates was successful a declarative
```

Comment:
Also to determine what/whether the update that needs to be performed in the first place

Reply:
See wording now